sensor_msgs/CompressedImage image
geometry_msgs/Pose pose